Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -22,6 +23,17 @@ public class TapToPlace : MonoBehaviour, IInputClickHandler
         [Tooltip("Supply a friendly name for the anchor as the key name for the WorldAnchorStore.")]
         public string SavedAnchorFriendlyName = "SavedAnchorFriendlyName";
 
+        [Tooltip("Place parent on tap instead of current game object.")]
+        public bool PlaceParentOnTap = false;
+
+        [Tooltip("Specify the parent game object to be moved on tap, if the immediate parent is not desired.")]
+        public GameObject ParentGameObjectToPlace;
+
+        /// <summary>
+        /// Keeps track of if the user is moving the object or not.
+        /// </summary>
+        public bool IsBeingDragged;
```

Comment:
@HodgsonSDAS Looking back I realized I actually said `IsBeingPlaced` instead of `IsBeingDragged` ... I think `IsBeingPlaced` is actually the best choice. 😄